### ISSUE TITLE
Sirius web Node Tool displays blank grey message box

### ISSUE THREAD

**Issue by sezginaytac** [eclipse-sirius/sirius-web#212]
Hi :)
I want to use Node Tool in Sirius Web. I downloaded current sample app jar, and created new schema, then followed the Stickies example demonstrated in Melanie' s youtube video.  Lastly, I created Node Tool for Member domain object. I have attached the image for the definitions. When trying the tool a blank gray toaster displayed and no message displayed in the application console. I think I am missing some point, but I could' t figure out. If I could be successful then I will try create a composition of objects in a single action. 
Many thanks in advance for your interest.
Best regards,

![sirius-web-node-tool-issue-001](https://user-images.githubusercontent.com/19195055/164248544-f1c7b804-3ee4-4a8d-8303-40a3bcc21e05.png)

The message box:
![sirius-web-node-tool-issue-002](https://user-images.githubusercontent.com/19195055/164248672-a3d6eafa-d2b4-4845-b4d2-5adca7bfb8b1.png)

**Comment by mbats** [comment]
Hi,
I tried to reproduce your sample with the online demo, you can find the Studio definition here : https://demo.obeostudio.com/projects/13b06f11-1985-4be2-b97b-163e2c2d8323/edit
And an instance here:
https://demo.obeostudio.com/projects/5213d03e-6a39-43ae-a039-ab7ea52d79de/edit/35d4e7c3-8ca1-4793-bf73-09869aa86ff9

I did not reproduce your error, could you check if there is some differences between what you did and what I create in the View definition?

Do you have some error log visible on the server side?

Best regards,

**Comment by sezginaytac** [comment]
Thanks much for your kind interest. Since I don' t have Obeo Cloud account, I am using the Sirius Web Sample Application, so I could not get the definition and instance you created. No error log on the server side 
Best regards,

**Comment by mbats** [comment]
You can create one yourself by signing up on : https://demo.obeostudio.com/signup
In order that you can compare what I did with your definition, I attached also the domain & view project to this comment:
[StickiesStudio.zip](https://github.com/eclipse-sirius/sirius-web/files/8529575/StickiesStudio.zip)

**Comment by sezginaytac** [comment]
Very very special thanks for your quick interest and response. 
By looking your definition I changed the "Type Name" definition as "stickies::Member" instead of "Member", then it worked :)
One small problem I could' t be overcome. I couldn' t  use "Set Value" on that new instance. 
For "Fetaure Name" definition, I have tried

- name
- aql:self.name
- aql:newInstance.name

How can I set the "name" field to a value?

Thank much again and best regards,


![sirius-web-node-tool-issue-003](https://user-images.githubusercontent.com/19195055/164427372-f8b40779-de46-455a-b678-3597f2e47352.png)
